scripts/test-html.js

```diff
@@ -0,0 +1,77 @@
+const process = require("process");
+const { JSDOM } = require("jsdom");
+const path = require("path");
+const http = require("http");
+const static = require('node-static');
+
+const { dir, base: filename } = path.parse(process.argv[2]);
+
+console.log(`serving: ${dir}`);
+
+serveDirectory(dir).then(server => {
+  const url = makeURL(server.address(), filename);
+
+  console.log(`loading ${url}`);
+
+  JSDOM.fromURL(url, {
+    resources: "usable",
+    runScripts: "dangerously",
+  })
+    .then(waitComplete)
```

Comment:
This formatting is a bit weird, and does not match what you use a few lines below for `.catch`. Consider
```suggestion
  }).then(waitComplete)
```